tests/integration-test.sh

```diff
@@ -22,19 +22,21 @@ dpkg -l > package.list
 
 
 buildinfo=$(drone build info vmware/vic $DRONE_BUILD_NUMBER)
+prNumber=$(drone build info --format "{{ .Ref }}" vmware/vic $DRONE_BUILD_NUMBER | cut -f 3 -d'/')
+prBody=$(curl https://api.github.com/repos/vmware/vic/pulls/$prNumber | jq -r ".body")
 
 if [[ $DRONE_BRANCH == "master" || $DRONE_BRANCH == "releases/"* ]] && [[ $DRONE_REPO == "vmware/vic" ]] && [[ $DRONE_BUILD_EVENT == "push" ]]; then
     echo "Running full CI for $DRONE_BUILD_EVENT on $DRONE_BRANCH"
     pybot --removekeywords TAG:secret --exclude skip tests/test-cases
 elif [[ $DRONE_BRANCH == *"refs/tags"* ]] && [[ $DRONE_REPO == "vmware/vic" ]] && [[ $DRONE_BUILD_EVENT == "tag" ]]; then
     echo "Running only Group11-Upgrade and 7-01-Regression for $DRONE_BUILD_EVENT on $DRONE_BRANCH"
     pybot --removekeywords TAG:secret --suite Group11-Upgrade --suite 7-01-Regression tests/test-cases
-elif grep -q "\[full ci\]" <(drone build info vmware/vic $DRONE_BUILD_NUMBER); then
+elif grep -q "\[full ci\]" <$prBody; then
     echo "Running full CI as per commit message"
     pybot --removekeywords TAG:secret --exclude skip tests/test-cases
-elif (echo $buildinfo | grep -q "\[specific ci="); then
+elif (echo $prBody | grep -q "\[specific ci="); then
     echo "Running specific CI as per commit message"
-    buildtype=$(echo $buildinfo | grep "\[specific ci=")
+    buildtype=$(echo $prBody | grep "\[specific ci=")
```

Comment:
I think it would be good if this can also run multiple specific groups if someone wants to run more than one groups.

Reply:
@anchal-agrawal has commented on this in the past. This actually gets passed in as a raw string, so if commas or the pybot delimiter is put in properly it already can. correct me if I am wrong @anchal-agrawal :)

Reply:
Correct! Multiple groups/suites can be specified by saying, for instance, `[specific ci=foo bar]`.